src/main/java/vic/cinema/dto/OrderResponseDto.java

```diff
@@ -0,0 +1,12 @@
+package vic.cinema.dto;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class OrderResponseDto {
+    private Long userId;
+    private LocalDateTime orderDate;
+    private List<TicketResponseDto> tickets;
```

Comment:
```suggestion
    private List<Long> ticketIds;
```
in case of any information required, and additional request may be made passing the ticket id

Reply:
Done